docs/guides/serverless-shared-migration-guide.md

```diff
@@ -0,0 +1,242 @@
+---
+page_title: "Migration Guide: Transition out of Serverless Instances and Shared-tier clusters"
+---
+
+# Migration Guide: Transition out of Serverless Instances and Shared-tier clusters
+
+The goal of this guide is to help users transition from Serverless Instances and Shared-tier clusters (M2/M5) to Free, Flex or Dedicated Clusters. 
+
+After March 2025, all Shared-tier clusters (`mongodbatlas_cluster` and `mongodbatlas_advanced_cluster`) will automatically convert to Flex clusters and all Serverless instances (`mongodb_serverless_instance`) will be converted into Free/Flex/Dedicated clusters, resulting in configuration drift in Terraform. 
+
+
+You can migrate from Serverless instances and Shared-tier clusters manually before March 2025. 
+
+**NOTE:** We recommend waiting until March 2025 for Serverless instances and Shared-tier clusters to autoconvert.
```

Comment:
can we repeat this into every "pre-autoconversion" section?

Reply:
Done